docs/contribute.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Primer
+
+There are several ways you can contribute to and participate in Primer:
+
+- Participate in discussions
+- Submit an issue
+- Improve the documentation
+- Close issues
+- Propose a new component or UI pattern
+- Questions and support
+
+Please note that all contributions to Primer need to follow our code of conduct.
+
+## Participate in discussions
+
+Discussions about improvements and upcoming work happen mostly in issues and in the #design-system Slack channel. These are the repos you should watch if you’re interested in getting involved:
+
+- Design
+- Primer CSS
+- Primer Components (React)
+- Octicons
+- ...
+
+The Design Systems Team holds office hours twice a week and anyone at GitHub is free to join, ask questions and participate in ongoing conversations.
```

Comment:
Instead of sending a calendar invite to everyone, could we here refer to the channel where the meetings start?

Reply:
Yeah, we could link to #design-systems. I have heard from folks that they aren't aware of the meetings because they aren't on the calendar, but we did that intentionally to avoid noise